osclib/repair_command.py

```diff
@@ -0,0 +1,59 @@
+import re
+
+from osc import oscerr
+from osc.core import change_review_state
+from osc.core import get_request
+from osclib.request_finder import RequestFinder
+
+def _is_int(x):
+    return isinstance(x, int) or x.isdigit()
+
+class RepairCommand(object):
+
+    def __init__(self, api):
+        self.api = api
+
+    def repair(self, request):
+        reviews = []
+        reqid = str(request)
+        req = get_request(self.api.apiurl, reqid)
+
+        if not req: 
+            raise oscerr.WrongArgs('Request {} not found'.format(reqid))
+
+        if req.state.name != 'review':
+            print('Request "{}" is not in review state'.format(reqid))
```

Comment:
Nice, but to avoid some corner cases is better to add `from __future__ import with_statement` in the first line